pkg/broker/filter/filter_handler.go

```diff
@@ -48,6 +47,12 @@ const (
 
 	// readyz is the HTTP path that will be used for readiness checks.
 	readyz = "/readyz"
+
+	// TODO make these constants configurable (either as env variables, config map, or part of broker spec).
+	// Constants for the underlying HTTP Client transport. These would enable better connection reuse.
+	// Set them on a 10:1 ratio, but this would actually depend on the Triggers' subscribers and the workload itself.
+	defaultMaxIdleConnections        = 1000
```

Comment:
The comment about the chosen values should be added here too.

Reply:
Done, thanks!